frontend/src/assets/css/globalStyles.css

```diff
@@ -128,7 +128,10 @@ form.inquirer-gui .theme--light:not(.v-item--active) .v-icon {
   color: var(--vscode-input-foreground, white);
 }
 
-form.inquirer-gui div.v-card{
+form.inquirer-gui div.theme--light.v-card.v-card--outlined {
+  border-width: thin;
+  border-style: solid;
+  border-color: var(--vscode-settings-textInputBorder, #cecece);
```

Comment:
vscode-settings-textInputBorder not supported by Theia yet.

![image](https://user-images.githubusercontent.com/57438361/76497908-8fb85f00-6444-11ea-9287-64a0dccf2e0c.png)

Reply:
What do you propose?